### ISSUE TITLE
Incorrect role is defined for the ‘Snooze’ and ‘dismiss’ control.

### ISSUE THREAD

**Issue by 232amansingh** [microsoft/BotFramework-WebChat#5750]
### Is it an issue related to Adaptive Cards?

Yes, this is related to Adaptive Card. However, it does not repro on Adaptive Cards Designer.

### What is the PWD impact?

When an incorrect role is defined for the ‘Snooze’ and ‘dismiss’ controls, screen reader users receive misleading or incomplete information. For example, if these controls are marked as role="link" but actually behave like buttons, users may expect navigation instead of an action.

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

Repro Steps: ​           
1. Open URL: Web Chat in edge browser.
2.Ask query 'Card Breakfast'.
3. Navigate to ‘Place to buy’ and ‘Related product’ control. 
4. Observe the issue that Incorrect role is defined for the ‘Snooze’ and ‘dismiss’ control.

### What do you expect?

Role as button should be defined for the ‘Snooze’ and ‘dismiss’ control. 

### What actually happened?

Incorrect role is defined for the ‘Snooze’ and ‘dismiss’ control.

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/a7e6b544-3088-4396-8adc-18a23734205a

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

### Adaptive Card JSON

```json

```
